test/Dialect/Torch/verify-backend-contract-unimplemented-op.mlir

```diff
@@ -0,0 +1,10 @@
+// RUN: torch-mlir-opt -torch-verify-backend-contract -split-input-file -verify-diagnostics %s
+func.func @forward(%arg0: !torch.vtensor<[3,5],f32>) -> !torch.vtensor {
+  %none = torch.constant.none
+  %0 = torch.tensor_static_info_cast %arg0 : !torch.vtensor<[3,5],f32> to !torch.vtensor<*,f32>
+  %1 = torch.copy.to_tensor %0 : !torch.tensor<*,f32>
+  // expected-error @+1 {{unsupported by backend contract: Unimplemented operator 'aten.clamp.Tensor'}}
+  %2 = torch.operator "aten.clamp.Tensor"(%1, %1, %none) : (!torch.tensor<*,f32>, !torch.tensor<*,f32>, !torch.none) -> !torch.tensor
```

Comment:
Is `aten.clamp.Tensor` a real operation that might be implemented one day? If so, would it be legal to use a made up op name e.g. `an.unimplemented.op`?

Reply:
Good idea, I will adapt